jolly-jellyfish/src/page_maker/templates/registration/register.html

```diff
@@ -0,0 +1,17 @@
+{% extends "page_maker/base.html" %}
+{% block content %}
+   <div>
+       <form method="POST">
+           {% csrf_token %}
```

Comment:
In most places we have this line 1 indent lower (to be the same level as the form before)